.github/renovate.json

```diff
@@ -209,6 +244,57 @@
         "@types/{/,}**"
       ]
     },
+    {
+      "extends": [
+        "monorepo:opentelemetry-js"
+      ],
+      "matchUpdateTypes": [
+        "patch",
+        "minor"
+      ],  
+      "groupName": "opentelemetry-js (non-major)",
+      "automerge": false      
+    },
+    { 
+      "extends": [
+        "monorepo:opentelemetry-js"
+      ],
+      "matchUpdateTypes": [
+        "major"
+      ], 
+      "groupName": "opentelemetry-js",
+      "automerge": false      
+    },
+    {
+      "extends": [
+        "monorepo:opentelemetry-js-contrib"
+      ],
+      "matchUpdateTypes": [
+        "patch",
+        "minor"
+      ],  
+      "groupName": "opentelemetry-js-contrib (non-major)",
+      "automerge": false      
+    },
+    { 
+      "extends": [
+        "monorepo:opentelemetry-js-contrib"
+      ],
+      "matchUpdateTypes": [
+        "major"
+      ], 
+      "groupName": "opentelemetry-js-contrib",
+      "automerge": false      
+    },    
+    {
+      "matchUpdateTypes": [
+        "major"
+      ],
+      "groupName": "types ",
+      "matchPackageNames": [
+        "@types/{/,}**"
+      ]
+    },
```

Comment:
Looks like this is a duplicate of lines 238-246

Reply:
yes, looks like I made a copy and paste error in an effort to reorg